torch/utils/tensorboard/pytorch_graph.py

```diff
@@ -0,0 +1,274 @@
+import numpy as np
```

Comment:
Is this module public? If it is, what does it do? If it is not, please prefix with an underscore.

Reply:
This converts a PyTorch graph to the proto format, but is internal. Prefixed.